app/src/main/java/com/khmelenko/lab/travisclient/adapter/JobsListAdapter.java

```diff
@@ -60,8 +60,7 @@ public void onBindViewHolder(JobViewHolder holder, int position) {
 
                 Drawable drawable = BuildStateHelper.getBuildImage(state);
                 if (drawable != null) {
-                    drawable.setBounds(0, 0, 30, 30);
```

Comment:
It might affect on the position of the drawable if we are not setting bounds. Did you check that?